app/logic/invitation_texts.rb

```diff
@@ -169,7 +169,8 @@ def first_response_conditions(protocol_completion, _curidx)
     end
 
     def current_index(protocol_completion)
-      protocol_completion.find_index { |entry| entry[:future] }
+      # || 0 in case there are no other measurements
+      protocol_completion.find_index { |entry| entry[:future] } || 0
```

Comment:
Komt dit ook voor in de praktijk? Bijv voor de laatste sms van de studie die ze krijgen? Is het dan niet beter om dan gewoon `protocol_completion.length` op te leveren? Volgens mij gebruiken we die current_index namelijk aleen om te kijken offie 0 is en dan de "kick-off" sms tekst te versturen (of alssie 1 is dan versturen we de "dit is je eerste dagboekvragenlijst").